front/app/api/custom_form/useCustomForm.ts

```diff
@@ -1,30 +1,30 @@
 import { useQuery } from '@tanstack/react-query';
 import { CLErrors } from 'typings';
 
+import { IPhaseData } from 'api/phases/types';
+
 import fetcher from 'utils/cl-react-query/fetcher';
 
+import getApiEndpoint from './getApiEndpoint';
 import customFormKeys from './keys';
-import { CustomFormKeys, ICustomForm, ICustomFormParameters } from './types';
+import { CustomFormKeys, ICustomForm } from './types';
 
-const fetchCustomForm = ({ projectId, phaseId }: ICustomFormParameters) => {
-  const apiEndpoint = phaseId
```

Comment:
In the past, we would not have a `phaseId` if `isFormPhaseSpecific` was false, which is only the case when the participation method is `ideation` (when we use `ideationConfig` as the form builder config). So now, we check that directly inside `getApiEndpoint` instead.